salt/proxy/junos.py

```diff
@@ -113,6 +113,15 @@ def init(opts):
         thisproxy['initialized'] = False
         return
 
+    if 'timeout' in proxy_keys:
+        timeout = int(opts['proxy']['timeout'])
+        try:
+            thisproxy['conn'].timeout = timeout
+        except Exception as ex:
+            log.error('Not able to set timeout due to: {}'.format(ex))
+        else:
+            log.debug('RPC timeout set to {} seconds'.format(timeout))
```

Comment:
Same as above:
```
log.debug('RPC timeout set to %d seconds', timeout)
```

Reply:
Do you mean `format` function call cannot be used in logging ??

Reply:
Yes.
It can, of course, but it shouldn't for performance reasons.

Reply:
See the fixed log calls on my commends.